packages/core/strapi/lib/core-api/controller/index.js

```diff
@@ -18,7 +18,7 @@ const createController = ({ contentType }) => {
       return transformResponse(data, meta, { contentType });
     },
 
-    sanitizeOutput(data, ctx) {
```

Comment:
Is this needed?

Reply:
yes you should always mark a function as async if it return a call to an async function this avoids any possible bug the day to decide to return a non promise and your code was doing xx.then() the async ensure a promise will always be returned